README.md

```diff
@@ -1,2 +1,20 @@
 # pdffonts
-Node bindings for Poppler's pdffonts CLI
+
+Node bindings for Poppler's `pdffonts` CLI.
+
+## Dependencies
+
+For this module to install and build correctly, you'll need to make sure that [`poppler`](https://poppler.freedesktop.org/) is installed on your machine.
```

Comment:
Might want to add that you can install with the command below, just for ease of use:

```
brew install poppler
```